adoc/admin-troubleshooting.adoc

```diff
@@ -57,6 +95,22 @@ For example, if bootstraps the first master node of the cluster fails, re-run th
 skuba node bootstrap --user sles --sudo --target <IP/FQDN> <NODE NAME> -v=1
 ----
 
+However if the bootstrap or join procedure fails at the last final steps, re-running it might not help, giving you an error similar to this:
+
+[source,bash]
+----
+error execution phase control-plane-join/update-status: error uploading configuration: unable to create ConfigMap: etcdserver: request timed out
```

Comment:
What if instead of referring to this esoteric message we describe that removing the node needs to be done if `kubectl get nodes` shows the node on the list.